app/views/govuk_publishing_components/components/_figure.html.erb

```diff
@@ -3,26 +3,27 @@
   alt ||= ""
   caption ||= ""
   credit ||= ""
+  src ||= nil
   local_assigns[:lang] ||= "en"
   component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
   component_helper.add_class("gem-c-figure")
 %>
-<%= tag.figure(**component_helper.all_attributes) do %>
-  <% if src.present? %>
+<% if src.present? && !src.blank? %>
```

Comment:
I think we can just use `src.present?` here as I believe it checks against `nil`, empty strings, and whitespace-only strings.

Reply:
Good spot, I've made this change now, thanks @JamesCGDS